README.md

```diff
@@ -145,41 +145,50 @@ issue. We thank you in advance for your cooperation.
 
 ### Requirements
 
-* C compiler
-* [Go compiler](https://golang.org/dl/)
-* liblzma-dev, libssl-dev and libglib2.0-dev packages
-
-#### LZMA support opt-out
-
-If no LZMA Artifact compression support is desired, you can ignore the `liblzma-dev` package
-dependency and substitute the `make` commands in the instructions below for:
+* C++ compiler
+* cmake
+* libarchive-dev, libboost-all-dev, liblzma-dev and libssl-dev packages
 
+For Debian/Ubuntu, the prerequisites can be installed by
 ```
-make TAGS=nolzma
+sudo apt install git build-essential cmake libarchive-dev liblmdb-dev libboost-all-dev libssl-dev
 ```
+Adjust as needed for other distributions.
 
-#### D-Bus support opt-out
 
-If no D-Bus support is desired, you can ignore the `libglib2.0-dev` package dependency and
-substitute the `make` commands in the instructions below for:
+### Steps
+
+To install Mender C++ client on a device from source, first clone the repository:
 
 ```
-make TAGS=nodbus
+git clone https://github.com/mendersoftware/mender.git -b feature-c++-client
 ```
 
-### Steps
+Change into the cloned repository:
+```
+cd mender
+```
 
-To install Mender on a device from source, first clone the repository in the correct folder
-structure inside your `$GOPATH` (typically `$HOME/go`):
+Use `git submodule` to fetch additional dependencies:
+```
+git submodule update --init --recursive
+```
 
+Create a build directory and enter it:
 ```
-git clone https://github.com/mendersoftware/mender.git $GOPATH/src/github.com/mendersoftware/mender
+mkdir build
+cd build
```

Comment:
Do we need the `build` approach? Less is more, is what I'm thinking.

Reply:
Out-of-tree builds are a best practice as far as I know, and it also matches the procedures by Yocto and friends closer. So I'd like to keep it. But your call, of cause.

Reply:
If you tested it, it's fine. I agree that out-of-tree builds are better, but we aren't testing them much.